lib/muser_priv.h

```diff
@@ -36,12 +36,14 @@
 #include "muser.h"
 
 int
-muser_pci_hdr_access(lm_ctx_t * const lm_ctx, size_t * const count,
-                     loff_t * const pos, const bool write,
-                     unsigned char *const buf);
+muser_pci_hdr_access(lm_ctx_t *lm_ctx, size_t *count,
+                     loff_t *pos, bool write,
```

Comment:
This can also fit in two lines, considering the 80C width.

Reply:
Yes. I'll repost. The order of these variables is not very natural to me. Maybe we should rearrange them altogether.